tests/test_ingest_files.py

```diff
@@ -1,13 +1,26 @@
 import unittest
 import sys
+from unittest.mock import patch
 
 sys.path.append("../ingest")
 from ingest_files import IngestFiles
 
 
 class TestIngestFiles(unittest.TestCase):
+    # Test exception
+
+    def raise_error(*args):
+        raise Exception
+
     def test_is_gzipped(self):
-        zipped_path = "../tests/data/mtx/AB_toy_data_toy.unsorted_mtx.mtx.gz"
+        zipped_path = "../tests/data/mtx/unsorted_matrix.mtx.gz"
+
         self.assertTrue(IngestFiles.is_gzipped(zipped_path))
+        gzip_extension = "../tests/data/mtx/unsorted_mtx.txt.gzip"
```

Comment:
`unsorted_mtx.gzip` would be more representative than `unsorted_mtx.txt.gzip`, right?

```suggestion
        gzip_extension = "../tests/data/mtx/unsorted_mtx.gzip"
```